src/pages/Teams.tsx

```diff
@@ -4,41 +4,57 @@ import {getTeams as fetchTeams} from '../api';
 import Header from '../components/Header';
 import List from '../components/List';
 import {Container} from '../components/GlobalComponents';
+import SearchBar from '../components/SearchBar';
 
-var MapT = (teams: TeamsList[]) => {
+const mapTeamsToItems = (teams: TeamsList[]) => {
     return teams.map(team => {
-        var columns = [
-            {
-                key: 'Name',
-                value: team.name,
-            },
-        ];
         return {
             id: team.id,
             url: `/team/${team.id}`,
-            columns,
+            columns: [
+                {
+                    key: 'Name',
+                    value: team.name,
+                },
+            ],
             navigationProps: team,
         } as ListItem;
     });
 };
 
 const Teams = () => {
-    const [teams, setTeams] = React.useState<any>([]);
-    const [isLoading, setIsLoading] = React.useState<any>(true);
+    const [teams, setTeams] = React.useState<TeamsList[]>([]);
```

Comment:
Changing initial states to match the correct type they are representing instead of generic values "any"  and any[]